apps/chat-e2e/src/tests/ChatSwitch.test.ts

```diff
@@ -0,0 +1,160 @@
+import { Conversation } from '@/chat/types/chat';
```

Comment:
let's follow lower-case naming pattern for .ts files, please rename ChatSwitch.test.ts -> chatSwitch.test.ts

Reply:
done